PowerPointLabs/PowerPointLabs/ActionFramework/Action/FrostedGlassActionHandler.cs

```diff
@@ -0,0 +1,20 @@
+﻿using PowerPointLabs.ActionFramework.Common.Attribute;
+using PowerPointLabs.ActionFramework.Common.Extension;
+using PowerPointLabs.ActionFramework.Common.Interface;
+
+namespace PowerPointLabs.ActionFramework.Action
+{
+    [ExportActionRibbonId("frostedGlassEffect")]
+    class FrostedGlassActionHandler : ActionHandler
```

Comment:
Since FrostedGlass is a sub-feature of Effects Lab,
I prefer the naming `EffectsLabFrostedGlass` instead of `FrostedGlass`.